### ISSUE TITLE
Regex contract needs instance

### ISSUE THREAD

**Issue by tirthb** [gnidan/solregex#3]
Currently, the regex contract needs an instance. It would be more convenient to not use it without instance. This is the error we get when we use matches without instance.
```
browser/TestRegex.sol:8:16: TypeError: Member "matches" not found or not visible after argument-dependent lookup in type(contract DobRegex)
        return DobRegex.matches(dob);
               ^--------------^
```

**Comment by gnidan** [comment]
Just published changes in v0.3.1 that should address this by outputting a `library` instead of a `contract`.

@tirthb not sure if you're still using this tool; let me know if you are and if this helps!

**Comment by tirthb** [comment]
Thanks @gnidan. I am not actively developing in blockchain anymore.
